src/Bicep.Cli/Commands/BuildParamCommand.cs

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Bicep.Cli.Arguments;
+using Bicep.Cli.Logging;
+using Bicep.Cli.Services;
+using Bicep.Core;
+using Bicep.Core.Parsing;
+using Bicep.Core.Syntax;
+using Bicep.Core.FileSystem;
+using Microsoft.Extensions.Logging;
+using Bicep.Core.Diagnostics;
+
+namespace Bicep.Cli.Commands
+{
+    public class BuildParamCommand
```

Comment:
Can we add a note to mention that this is temporary and will be merged into `BuildCommand`?